proto/sample_v2_round_trip.json

```diff
@@ -3614,11 +3619,10 @@
             "warningAuditsGroupTitle": "Passed audits but with warnings", 
             "warningHeader": "Warnings: "
         }
-    },
+    }, 
     "lighthouseVersion": "4.3.0", 
     "requestedUrl": "http://localhost:10200/dobetterweb/dbw_tester.html", 
-    "runWarnings": [],
-    "stackPacks": [],
```

Comment:
this happened on update:sample-json, is that bad @exterkamp ?

Reply:
based on appveyor it appears the answer is, yes this is bad :)

Reply:
Yes this is bad :smile: That is uh, slightly broken right now.  Revert the stack packs line.

Reply:
this is #8245

Reply:
oh right I even commented on that 😆 it's been a week...